### PR TITLE
update python textanalytics for preview 2

### PR DESCRIPTION
**Pull request by kristapratico**

### CODE REVIEW

File: releases/2020-02/python.md [hunk 1]
```diff
@@ -55,6 +55,15 @@ Detailed change logs are linked to in the Quick Links below. Here are some criti
 
 - This release contains bug fixes to improve quality.
 
+### Text Analytics
+
```
Comment (samvaity) [inline releases/2020-02/python.md]: Should you include a link for the detailed changelog?

Comment (kristapratico) [reply releases/2020-02/python.md]: Is this something new we're doing? I don't see it in the release notes guidelines -  https://azure.github.io/azure-sdk/policies_releasenotes.html

Comment (kaerm) [reply releases/2020-02/python.md]: @adrianhall should changelog be added to the guidelines for release notes?

Comment (maririos) [reply releases/2020-02/python.md]: I got that direction from Adrian in my PR. See https://github.com/Azure/azure-sdk/pull/1005#discussion_r377760060

Comment (kristapratico) [reply releases/2020-02/python.md]: Thanks, added

**Pull request merged**